ext/css/display.css

```diff
@@ -644,6 +644,9 @@ button.action-button[hidden] {
         -webkit-filter var(--animation-duration) linear,
         background-color var(--animation-duration) linear;
 }
+button.collapsable-action-button[hidden] {
```

Comment:
This is a super minor nitpick, but change `collapsable` to `collapsible` (using an "i"). Search the code for both words and you'll see the one with the "i" is mainly used; there are a few other cases with the "a" that I'll fix up soon.